### ISSUE TITLE
Edit subject not available in open message window

### ISSUE THREAD

**Issue by PissedFirefoxUser** [cleidigh/EditEmailSubject-MX#42]
Just discovered that the right click menu does not show Edit Mail Subject when clicked in open message window, but does show in main windows when message is selected. It used to be available always.

**Comment by dumb0x** [comment]
I agree that this is a surprising deficiency (and was about to report it before finding this comment), as the message window is the place most appropriate for changing the Subject: line.  Is that intended? (TB78.6.1/W10/64)

**Comment by TossOne53** [comment]
Hello, long time user and miss the ability to change the subject while in the open message window.  Do you know:
1.  Given TB 78+, is it possible to add the ability in an open message window?
2.  If so, is there a plan to do so?
Thank you, I'm on TB 78.8.1 and Win10 if that is of interest.

**Comment by bege10** [comment]
Yes, that is a pity, and the hotkey does not work either.

**Comment by dvschuetz** [comment]
on Windows 10 Pro 20H1 with TB 78.14.0 (both German, 64bit) and EES 4.4 same issue: 
does not work in message windows (and does not appear in main menu). 
Which was both the case until version 2.1 (until TB 59).
Anyway, thank you all for converting this old but extremely helpful AddOn.
